Automattic/jetpack
# Jetpack search: Add default sort by price

## Proposed changes:
Currently it is only possible to set default sort options to newest, oldest, and relevance. Some WooCommerce users would like to be able to also specify a default sort by price or rating. This adds those options to the UI for the Jetpack Search settings.

### Other information:



## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
To test, spin up a new site with this branch. Look at the Jetpack Search settings. You should see the option to set the default sort to price and rating. Choose price, and then perform a search, and check that it is sorted by price by default.
